capella2polarion/converters/element_converter.py

```diff
@@ -390,6 +395,30 @@ def _get_requirement_types_text(
             type_texts[req.type.long_name].append(req.text)
         return _format_texts(type_texts)
 
+    def _add_diagram(
+        self,
+        converter_data: data_session.ConverterData,
+        diagram_attr: str,
+        diagram_label: str,
+        render_params: dict[str, t.Any] | None = None,
+        filters: list[str] | None = None,
+    ) -> data_model.CapellaWorkItem:
+        """Add a new custom field diagram based on provided attributes."""
+        assert converter_data.work_item, "No work item set yet"
+        diagram = getattr(converter_data.capella_element, diagram_attr)
+
+        for filter in filters or []:
+            diagram.filters.add(filter)
+
+        self._draw_additional_attributes_diagram(
+            converter_data.work_item,
+            diagram,
+            diagram_attr,
```

Comment:
I would recommend to have an additional parameter for this as you pass the diagram attribute as custom field id. This will be the same in most cases, but especially later when we also want to define custom diagrams and maybe have multiple of those, this would need an adjustment anyway